cloudwatch.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_cloudwatch_log_group" "codebuild_log_group" {
+  name              = "/aws/codebuild/${local.project_name_codebuild}"
+  retention_in_days = 90  # Log retention set to 90 days
```

Comment:
Make the log retention a variable (also no need for a comment that's basically the argument name + value ;) )